src/test/java/uk/ac/ebi/eva/pipeline/io/mappers/VariantVcfFactoryTest.java

```diff
@@ -49,17 +49,17 @@
     public void testRemoveChrPrefixInAnyCase() {
```

Comment:
Not part of this ticket but can you please modify this name. There was a change in a previous ticket but forgot to change the method name and is misleading right now. Thanks!
```suggestion
    public void testDontRemoveChrPrefixInAnyCase() {
```

Reply:
updated